core/commands/crypt.go

```diff
@@ -0,0 +1,120 @@
+package commands
+
+import (
+	cmds "github.com/ipfs/go-ipfs/commands"
+	crypt "github.com/ipfs/go-ipfs/crypt"
+)
+
+var CryptCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline:          "perform ipfs cryptographic operations",
+		ShortDescription: ``,
+	},
+	Subcommands: map[string]*cmds.Command{
+		"en": encryptCmd,
+		"de": decryptCmd,
+	},
+}
+
+var encryptCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline:          "encrypt data using ipfs keypairs",
+		ShortDescription: ``,
+	},
+	Arguments: []cmds.Argument{
+		cmds.FileArg("data", true, false, "data to encrypt").EnableStdin(),
+	},
+	Options: []cmds.Option{
+		cmds.StringOption("k", "key", "key to use for encryption (default 'local')"),
+		cmds.StringOption("o", "output", "output file name (default stdout)"),
+		cmds.StringOption("p", "cipher", "cipher to use for encryption"),
+		cmds.StringOption("m", "mode", "block cipher mode to use for encryption"),
```

Comment:
This is incorrect. We want to implement proper AEAD. We already know the right ways to do these and it's not by letting users choose the mode.